tests/org.jboss.tools.examples.ui.bot.test/README.md

```diff
@@ -3,27 +3,44 @@ Executing tests:
 mvn clean install -P{profile} -Dswtbot.test.skip=true -Dusage_reporting_enable=false -DdeployOnServer=true
 
 Where
-	${profile} select maven profile: WildFly10, EAP7, JavaEE7, SmokeTestsEAP7, SmokeTestsWildFly10, SmokeTestsJavaEE7 (WildFly10 profile is selected, if you don't specify profile)
+	${profile} select maven profile: WildFly11, EAP7, JavaEE7, SmokeTestsEAP7, SmokeTestsWildFly10, SmokeTestsJavaEE7 (WildFly10 profile is selected, if you don't specify profile)
 	
 You can specify other parameters as well(depending on profile you have selected):
 WildFly profile:
+-PWildFly11
 -DquickstartsURLWildFly=${quickstartsURLWildFly}		-	URL for WildFly Quickstarts
 -DexamplesFolderWildFly=${examplesFolderWildFly}		-	examples folder of WildFly Quickstarts
 
+
+```
+mvn clean verify -DskipTests=false -DexamplesFolderWildFly=wildfly-11.0.0.Final-quickstarts -DspecificQuickstarts=jts -Dreddeer.close.welcome.screen=true -DquickstartsURLWildFly=https://github.com/wildfly/quickstart/releases/download/11.0.0.Final/wildfly-11.0.0.Final-quickstarts.zip
+```
+
 EAP profile:
+-PEAP7
 -DquickstartsURLEAP=${quickstartsURLEAP}				-	URL for EAP Quickstarts
 -DexamplesFolderEAP=${examplesFolderEAP}				-	examples folder of EAP Quickstarts
 -Djbosstools.test.jboss-eap-7.x.url=${EAP7_URL_OF_ZIP}	-	URL of zip file with EAP server
 
+G.e.:
+
+```
+mvn clean verify -DskipTests=false -Dusage_reporting_enabled=false -Dswt.bot.test.record.screencast=true -Dorg.eclipse.swtbot.screenshots.dir=./target/screenshots -Dcom.atlassian.connector.eclipse.monitor.usage.first.time=false -Dcom.atlassian.connector.eclipse.monitor.usage.enabled=false -Dmaven.settings.path=~/.m2/settings.xml -Drd.config=./target/classes/servers/eap-7.json -DdeployOnServer=false -Dreddeer.close.shells=true -Dreddeer.close.welcome.screen=true -Dreddeer.disable.maven.download.repo.index.on.startup=true -DquickstartsURLEAP=http://download.eng.brq.redhat.com/released/JBEAP-7/7.1.0/jboss-eap-7.1.0-quickstarts.zip -DexamplesFolderEAP=jboss-eap-7.1.0.GA-quickstarts -Djbosstools.test.jboss-eap-7.x.url=http://download.eng.brq.redhat.com/released/JBEAP-7/7.1.0/jboss-eap-7.1.0.zip -PEAP7 -DspecificQuickstarts=helloworld-jms
```

Comment:
Do we really need all of these parameters? Quite a lot of these parameters have default values in this pom or in parent poms.

Reply:
There were missing parameters for devstudios Debug/Run Configurations

Reply:
In this case you can also commit launch configurations(e.g. https://github.com/jbosstools/jbosstools-integration-tests/tree/master/tests/org.jboss.tools.forge.ui.bot.test/launchers) and leave here only necessary parameters. You are not specifying that you need all of these parameters when running from CLI or eclipse.